src/components/RegistrationForm/RegistrationForm.tsx

```diff
@@ -56,9 +57,9 @@ const RegistrationForm: React.FC<Props> = ({
   const history = useHistory();
 
   const formatConsentLabel = (label: string): string | JSX.Element => {
-    // @todo sanitize consent label to prevent XSS
-    const hasHrefOpenTag = /<a(.|\n)*?>/.test(label);
-    const hasHrefCloseTag = /<\/a(.|\n)*?>/.test(label);
+    const sanitizedLabel = DOMPurify.sanitize(label);
```

Comment:
Only `<a />` tags are allowed, perhaps limit the sanitizer to only accept that?

```suggestion
    const sanitizedLabel = DOMPurify.sanitize(label, { ALLOWED_TAGS: ['a'] });
```

Reply:
Hm, we get this value from the back-end side and it seems that other tags are also available to use.

Reply:
Tbh, I almost think we can skip sanitizing this at all... It's coming directly from what a company has put into their cleeng configurations. If they want to put a script or an iframe or something, that's up to them.

Reply:
Hmm, true. I guess the only way to misuse this is by doing a man-in-the-middle attack and somehow altering the response.

Reply:
Let's have some basic sanitizing here. In case we get requests from customers to remove it or understand that it is not the best solution, we will remove it.